tests/Integration.Test/Features/DicomDimseScp.feature

```diff
@@ -43,7 +43,7 @@ Feature: DICOM DIMSE SCP Services
         And <count> <modality> studies
         When a C-STORE-RQ is sent to 'Informatics Gateway' with AET '<aet>' from 'TEST-RUNNER'
         Then a successful response should be received
-        And <count> workflow requests sent to message broker
+        And <count> Artifact Recieved sent to ea message broker
```

Comment:
Are we replacing the Workflow Request completely with the Artifact Received event in this iteration?

Reply:
Do we want to keep the existing tests now if we are not replacing it now?